docs/internal-documentation/distrib-testing/Ubuntu-build-environment.md

```diff
@@ -17,12 +31,38 @@ This instruction is for creating it on Ubuntu. See other files for other OSs.
 3. Install prerequisites (binaries)
 
    ```sh
-   sudo apt-get install cpanminus gettext autoconf automake build-essential
+   sudo apt-get install cpanminus gettext autoconf automake build-essential libdevel-checklib-perl libmodule-install-xsutil-perl libssl-dev libidn11-dev libtool
    ```
 
-3. Install prerequisites (binaries)
+4. Install Docker - *only needed if Docker images are to be built*
 
    ```sh
-   sudo cpanm Module::Install
+   sudo apt-get install docker.io
```

Comment:
Maybe link to the Docker installation page instead https://docs.docker.com/engine/install/ubuntu/ ? (they start by removing the package `docker.io` if installed)

Reply:
What you get with `apt-get install docker.io`, I assume, is the Docker version in the Ubuntu repository. Even though it is not the latest version I am sure it will good enough for our use case. It gives Docker version 20.10.7 on Ubuntu 20.04. On my Mac where I have installed it directly from Docker I have version 20.10.8.

The installation following the URL you provided is not as straight forward. My compromise is to include the URL, but still have the `apt-get` command is the primary suggestion for installation. Would that be fine for you? Please see the update I have done.

Reply:
Yes I guess that would be alright.